frontends/election-manager/src/screens/tally_report_screen.tsx

```diff
@@ -184,6 +193,17 @@ export function TallyReportScreen(): JSX.Element {
     });
   }
 
+  function cancelConfirmingOfficial() {
+    setIsConfirmingOfficial(false);
+  }
+  function confirmOfficial() {
+    setIsConfirmingOfficial(true);
+  }
+  async function setOfficial() {
+    setIsConfirmingOfficial(false);
+    await saveIsOfficialResults();
+  }
```

Comment:
In terms of naming here, I see the two steps - 1) clicking to make results official, which opens a modal, and 2) confirming in the modal. `confirmOfficial()` makes me think of the second step when it's currently representing the first.

We could maybe make this more intuitive by renaming `isConfirmingOfficial` `isMarkResultsAsOfficialModalOpen`:

```ts
function openMarkResultsAsOfficialModal() {
  setIsMarkResultsAsOfficialModalOpen(true);
}
```

We could then rename `setOfficial` `markResultsAsOfficial`